lib/src/extensions.dart

```diff
@@ -254,6 +255,28 @@ extension ExpressionExtension on Expression? {
   bool get isNullLiteral => this?.unParenthesized is NullLiteral;
 }
 
+extension InhertanceManager3Extension on InheritanceManager3 {
```

Comment:
Uh...tiny bit confusing w/ the code move around in this PR.

Reply:
Yeah. I did a sort. (Which we generally try to do but are not currently enforcing.)

https://github.com/dart-lang/linter/issues/4273